packages/react-positioning/src/createArrowStyles.ts

```diff
@@ -65,15 +74,15 @@ export function createArrowStyles(size?: number): MakeStylesStyleRule<Theme> {
     },
 
     ':global([data-popper-placement^="right"])': {
-      left: 0,
+      [dir === 'ltr' ? 'left' : 'right']: 0,
       '--angle': '90deg',
     },
     ':global([data-popper-placement^="bottom"])': {
       top: 0,
       '--angle': '180deg',
     },
     ':global([data-popper-placement^="left"])': {
-      right: 0,
+      [dir === 'ltr' ? 'right' : 'left']: 0,
```

Comment:
This should flip automatically, did you check why it's not happening?

Reply:
The attribute on html is being flipped automatically! The problem is in our styles overrides, they're not flipping

Reply:
Oh see what you're meaning, yeah. that's actually weird 👀, perhaps a bug on the `:global` selector?

Reply:
Due to the usage of `useFluent` to programmatically decide the placement of the popper,

https://github.com/microsoft/fluentui/blob/be9d4afa0a0b36f4de36b2b3609d676ebaa424da/packages/react-positioning/src/usePopper.ts#L89

This means that if it's `RTL` the placement that was `left` becomes `right` and vice versa, and that conflicts with the inversion of `make-styles`.

So `make-styles` expects to find `:global([data-popper-placement^="left"])`  on the cases where the placement is in the `right` for `RTL`, but that won't be true.

Reply:
@bsunderhus I see, then `@noflip` should help:

`{ left: '5px /* @noflip */' }`

Can you please try?

Reply:
Turns out that this is a more complex problem to solve (although the current solution in this PR would be a quick fix). Basically the arrow positioning is decided by [arrow modifier](https://popper.js.org/docs/v2/modifiers/arrow) popper, and there's no option of providing a different direction to the arrow positioning mechanism provided by popper.

The real solution would be to stop using popper arrow positioning modifier and create our custom own that understands direction.

Reply:
Didn't know about this annotation @layershifter! thx this helped a lot actually.